app/controllers/workflows_controller.rb

```diff
@@ -58,29 +56,33 @@ def stage # rubocop:todo Metrics/CyclomaticComplexity
     # else actually execute the task.
     unless params[:next_stage].nil?
       eager_loading = @task.included_for_do_task
```

Comment:
This method is still pretty complicated, and handles both showing steps, and processing steps. I had hoped to split it into two distinct chunks, but my time-box ran out before I got there.

So changes are mostly concerned with letting tasks be editable. I've confined this to those tasks which I know work for the time being.